eZ/Publish/Core/FieldType/ImageAsset/Type.php

```diff
@@ -169,6 +169,14 @@ protected function checkValueStructure(BaseValue $value): void
                 $value->destinationContentId
             );
         }
+
+        if ($value->alternativeText !== null && !is_string($value->alternativeText)) {
```

Comment:
Mixed feelings here. Is there any other part of the system which sets that value not via `ImageAsset/Value constructor`?
Waiting impatiently for strictly typed class members...
Until we get them, maybe this is this one case where using private property and getter with strict return type is called for?

Reply:
> Is there any other part of the system which sets that value not via ImageAsset/Value constructor?

No,  but the `alternativeText` property is public (consistently with other classes representing FT value) so it's value needs to be validated. 

> Waiting impatiently for strictly typed class members...

Me too 😉